stroeer/core/v1/article.proto

```diff
@@ -961,6 +962,54 @@ message Body {
   repeated BodyNode children = 1;
 }
 
+// todo: rename to `Body`
+message BodyPart {
+  /*
+  * ### `types`
+  * | type | description |
+  * | -----| ------------|
+  * | `body` | formatted text and inlined elements |
+  * | `disclaimer` | formatted text that holds a disclaimer, e.g. used in `/gesundheit/` or `/kaufbertatung/` |
+  * | `infobox` | todo |
```

Comment:
infobox is an inline thing, so it should not be here?

Reply:
👍